overview.md

```diff
@@ -80,8 +80,27 @@ analytic database systems, data frame libraries, and more.
   <div class="col-md-12">
     <h3>Arrow Libraries</h3>
     <p>
-      The Arrow project contains libraries that enable you to work with data in the Arrow columnar format in many languages. The <a href="{{ site.baseurl }}/docs/cpp/">C++</a>, <a href="https://github.com/apache/arrow/blob/main/csharp/README.md">C#</a>, <a href="https://godoc.org/github.com/apache/arrow/go/arrow">Go</a>, <a href="{{ site.baseurl }}/docs/java/">Java</a>, <a href="{{ site.baseurl }}/docs/js/">JavaScript</a>, <a href="{{ site.baseurl }}/julia/">Julia</a>, and <a href="https://docs.rs/crate/arrow/">Rust</a> libraries
-      contain distinct implementations of the Arrow format. These libraries are <a href="{{ site.baseurl }}/docs/status.html">integration-tested</a> against each other to ensure their fidelity to the format. In addition, Arrow libraries for <a href="{{ site.baseurl }}/docs/c_glib/">C (Glib)</a>, <a href="https://github.com/apache/arrow/blob/main/matlab/README.md">MATLAB</a>, <a href="{{ site.baseurl }}/docs/python/">Python</a>, <a href="{{ site.baseurl }}/docs/r/">R</a>, and <a href="https://github.com/apache/arrow/blob/main/ruby/README.md">Ruby</a> are built on top of the C++ library.
+      The Arrow project contains libraries that enable you to work with data
+      in the Arrow columnar format in many languages. The
+      <a href="{{ site.baseurl }}/docs/cpp/">C++</a>,
+      <a href="https://github.com/apache/arrow/blob/main/csharp/README.md">C#</a>,
+      <a href="https://godoc.org/github.com/apache/arrow/go/arrow">Go</a>,
+      <a href="{{ site.baseurl }}/docs/java/">Java</a>,
+      <a href="{{ site.baseurl }}/docs/js/">JavaScript</a>,
+      <a href="{{ site.baseurl }}/julia/">Julia</a>,
+      <a href="https://docs.rs/crate/arrow/">Rust</a>,
+      and <a href="https://github.com/apache/arrow/blob/main/swift/README.md">Swift</a>
```

Comment:
This URL does not seem to exist:

https://github.com/apache/arrow/blob/main/swift/README.md

![Screenshot 2023-10-14 at 1 37 28 PM](https://github.com/apache/arrow-site/assets/490673/530ea230-2128-479e-a697-41d9c6058cb4)

Reply:
Oh, sorry...

```suggestion
      and <a href="https://github.com/apache/arrow/blob/main/swift">Swift</a>
```